patches/minecraft/net/minecraft/block/BlockBeacon.java.patch

```diff
@@ -0,0 +1,26 @@
+--- ../src-base/minecraft/net/minecraft/block/BlockBeacon.java
++++ ../src-work/minecraft/net/minecraft/block/BlockBeacon.java
+@@ -106,12 +106,12 @@
+ 
+     public static void func_176450_d(final World p_176450_0_, final BlockPos p_176450_1_)
+     {
++        Chunk chunk = p_176450_0_.func_175726_f(p_176450_1_); // Forge: don't access world off-thread
++
+         HttpUtil.field_180193_a.submit(new Runnable()
```

Comment:
I'd simply remove this block of runnable. See https://github.com/PaperMC/Paper/blob/459987d69fc6be722321dfbf42547613a62daaed/Spigot-Server-Patches/0172-Shame-on-you-Mojang.patch